lib/std/c/netbsd.zig

```diff
@@ -34,7 +34,15 @@ pub extern "c" fn __getrusage50(who: c_int, usage: *rusage) c_int;
 pub extern "c" fn __libc_thr_yield() c_int;
 
 pub extern "c" fn posix_memalign(memptr: *?*c_void, alignment: usize, size: usize) c_int;
-pub extern "c" fn malloc_usable_size(?*const c_void) usize;
+
+// malloc stats is no longer in libc (netbsd 9.1 places it in `libjemalloc`)
+usingnamespace if (false)
```

Comment:
did you mean to put if (netbsd) here?

Reply:
no, it's in a netbsd-only file. I just wanted some kind of reference that we are NOT using that fn because std.heap performs hasDecl() against it. I could do away with the usingnamespace/if and comment-out `malloc_usable_size` or simply clip the whole thing. Do you have a preference?